### DISCUSSION TITLE
Japanese Translation

### DISCUSSION THREAD

**Discussion by fujiken0201** [bafolts/terraforming-mars#6214] [Localization]
[pull/ja](https://github.com/terraforming-mars/terraforming-mars/pull/6213)

i issue pull requests for Japanese translations.
It is not yet merged, but if there are any Japanese users, it would be appreciated if you could confirm the translation.

**Comment by kberg** [comment]
There's been a Japanese translation for a while. I don't think this needs further discussion. Please comment if I am incorrect.
